krita.txt

```diff
@@ -0,0 +1,19 @@
+[Section]
+Name = Krita
+Version = 3.2.1
```

Comment:
```suggestion
Version = 3.2.1.0
```
To be more exact.

Reply:
In the software context, the webpage and the whole Krita context, that version is reported as 3.2.1, I will keep it like that. Searching in Google Krita 3.2.1.0 is not making any result.